src/main/java/team/themoment/hellogsmv3/domain/application/repo/custom/CustomApplicationRepositoryImpl.java

```diff
@@ -0,0 +1,38 @@
+package team.themoment.hellogsmv3.domain.application.repo.custom;
+
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import org.springframework.stereotype.Repository;
+import team.themoment.hellogsmv3.domain.application.entity.abs.AbstractApplication;
+
+import java.util.Optional;
+
+import static team.themoment.hellogsmv3.domain.application.entity.abs.QAbstractApplication.abstractApplication;
+import static team.themoment.hellogsmv3.domain.application.entity.abs.QAbstractMiddleSchoolGrade.abstractMiddleSchoolGrade;
+import static team.themoment.hellogsmv3.domain.application.entity.abs.QAbstractPersonalInformation.abstractPersonalInformation;
+import static team.themoment.hellogsmv3.domain.applicant.entity.QApplicant.applicant;
+
+
+
+
+@Repository
+public class CustomApplicationRepositoryImpl implements CustomApplicationRepository {
+
+    private final JPAQueryFactory queryFactory;
+
+    public CustomApplicationRepositoryImpl(JPAQueryFactory jpaQueryFactory) {
+        this.queryFactory = jpaQueryFactory;
+    }
+
+    @Override
+    public Optional<AbstractApplication> findByApplicantAuthenticationIdWithAllJoins(Long authenticationId) {
```

Comment:
```suggestion
    public Optional<AbstractApplication> findByAuthenticationIdWithAllJoins(Long authenticationId) {
```

Application 이라는 키워드는 클래스명에 있어 메서드에서는 생략하면 더 좋을것 같습니다.

Reply:
`Applicant`엔티티의 `authenticationId`로 찾는다는 의미로 `Applicant`를 메서드 네임에 포함시켰습니다.  
지워도 의미 전달이 잘 될 것 같아 보여서 수정하겠습니다.

a37007be489ca5b07b10f83d37dae4228fc380a5